cms/static/js/certificates/models/certificate.js

```diff
@@ -20,9 +20,25 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
         defaults: {
             name: 'Name of the certificate',
             description: 'Description of the certificate',
-            course_title: 'Title of the course',
+
+            /**
+             * It represents the course title.
+             */
+            course_title: '',
+
+            /**
+             * It represents the path to organization logo image.
+             */
             org_logo_path: '',
+
+            /**
+             * It represents the default version of the certificate.
```

Comment:
OK, thanks. Even though it's not user-facing, perhaps rewording to just 'version" rather than "default version" if I understand correctly?